pkg/nuctl/command/deploy.go

```diff
@@ -220,6 +221,7 @@ func newDeployCommandeer(ctx context.Context, rootCommandeer *RootCommandeer, be
 
 	addDeployFlags(cmd, commandeer)
 	cmd.Flags().BoolVarP(&commandeer.skipSpecCleanup, "skip-spec-cleanup", "", false, "Do not clean up spec in function configs")
+	cmd.Flags().BoolVarP(&commandeer.skipExternalRegistryVerify, "skip-external-registry-verify", "", false, "Do not verify if registry is external")
```

Comment:
I prefer the positive over the negative in this case:
```suggestion
	cmd.Flags().BoolVarP(&commandeer.skipExternalRegistryVerify, "verify-external-registry", "", false, "verify registry is external")
```

Reply:
@TomerShor so, we want default behaviour to be redeploying without registry check?

Reply:
Yes, since the registry check is relevant specifically for upgrade in an IGZ systems